modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/AppContext.es.js

```diff
@@ -138,24 +138,24 @@ const editFocusedCustomObjectField = ({
 	focusedCustomObjectField,
 	propertyName,
 	propertyValue,
-}) => {
-	let localizableProperty = false;
+}, editingLanguageId, dataLayoutBuilder) => {
+	let localizedValue;
 	const {settingsContext} = focusedCustomObjectField;
 	const visitor = new PagesVisitor(settingsContext.pages);
 	const newSettingsContext = {
 		...settingsContext,
 		pages: visitor.mapFields((field) => {
-			const {fieldName, localizable} = field;
+			const {fieldName} = field;
 
 			if (fieldName === propertyName) {
-				localizableProperty = localizable;
+				localizedValue = {
+					...field.localizedValue,
+					[editingLanguageId || themeDisplay.getLanguageId()]: propertyValue,
```

Comment:
Hey @MateusCastro @kevenleone 
Would it be possible to pass the default language of the field? I'm afraid that "themeDisplay.getLanguageId()" don't be the same in every situation. Example: fieldsets can have a different default language than the one returned by themeDisplay => fieldset was created when the language was English and the language of the instance was changed to Portuguese ...
What it is your opinion about that? I always prefer to look for the default language, if there is none for some reason, we can use themeDisplay :)

Reply:
Hey @marcelabc, thanks for this observation, you are right xD.

I'll do this change, and send asap.